src/frontend/components/ViewersList/StudentViewersList/index.tsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import { Box, List } from 'grommet';
```

Comment:
😱 and we lost the round effect in the snapshot aswell

Reply:
address above in this PR: 

> To create the border all around the avatar (a border only shown when the avatar is the one of an instructor), I was using the border props of grommet. But in this PR I stop to use this border props and use the outline css property instead. (I decided to switch border and outline because outline doesn't take any space, so it's more confortable to use in this precise case).